db/db_test.cc

```diff
@@ -10620,6 +10620,173 @@ TEST_F(DBTest, WalFilterTestWithChangeBatchExtraKeys) {
 
   ValidateKeyExistence(db_, keys_must_exist, keys_must_not_exist);
 }
+
+TEST_F(DBTest, WalFilterTestWithColumnFamilies) {
```

Comment:
@siying Does Linux now run db_test2 as a part of CI and should Windows build do the same?